ej1.rb

```diff
@@ -0,0 +1,17 @@
+class Long
+  attr_accessor :size
+  def initialize(size)
+    @size = size
+  end
+
+  def to_size
+    this = @size
```

Comment:
porque le pusiste `this` a la variable? no dice mucho acerca de que se guarda en ella